urth_dash_js/notebook/dashboard-common/error-log.js

```diff
@@ -1,43 +0,0 @@
-/**
- * Copyright (c) Jupyter Development Team.
- * Distributed under the terms of the Modified BSD License.
- */
-
-/**
- * Registers an error handler which prints errors to the JS console.
- */
-define([
-    'require',
-    'jquery'
-], function(
-    require,
-    $
-) {
-    'use strict';
-    var enabled = false;
-    return {
-        /**
-         * Show IOPub errors in the JS console.
-         * @param  {object} IPython    "global" IPython singleton
-         */
-        enable: function(IPython) {
```

Comment:
OK. It's good for widget folks to know this too, since they added a similar feature to what you just said jupyter-js-output-area now provides. /cc @lbustelo @wellecks